<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IngredientsSummaryForm</class>
 <widget class="QWidget" name="IngredientsSummaryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addIngredientButton">
       <property name="text">
        <string>Добавить новый продукт</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="ingredientNameInput">
       <property name="placeholderText">
        <string>Название продукта</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteIngredientButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="PFCInputHorizontalLayout">
     <item>
      <widget class="QLabel" name="proteinsLabel">
       <property name="text">
        <string>Белки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="proteinsInput">
       <property name="maxLength">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fatsLabel">
       <property name="text">
        <string>Жиры</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fatsInput">
       <property name="maxLength">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="carbohsLabel">
       <property name="text">
        <string>Углеводы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="carbohsInput">
       <property name="maxLength">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="ingredientCommitButton">
     <property name="text">
      <string>Подтвердить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
